ambassador/src/delegate_shared.rs

```diff
@@ -55,6 +56,9 @@ impl<T: DelegateTarget> DelegateArgs<T> {
                             let generics_val = lit.parse_with(Punctuated::<GenericParam, Comma>::parse_terminated).expect("Invalid syntax for delegate attribute; Expected list of generic parameters as value for \"generics\"");
                             res.generics.extend(generics_val);
                         }
+                        "inhibit_automatic_where_clause" => {
+                            res.inhibit_automatic_where_clause = true;
```

Comment:
Does this check to make the value was "true" (`inhibit_automatic_where_clause="true"`).  I don't think it should be enabled if someone has `inhibit_automatic_where_clause="false"`)

Reply:
Other than that, if you want to add documentation and tests, I'd be happy to merge this.